### PR TITLE
fix(jellyfish-api-core): Fix number of totalAuctions in getloaninfo RPC Ain#1124

### PR DESCRIPTION
**Pull request by chanakasameera**
kind/fix
#### What this PR does / why we need it:
Calculate totalAuctions by adding each liquidated vault's batch count instead of counting the number of liquidated vaults.
This will now match the number of open auctions displayed at https://defiscan.live/auctions
Ain#1124"

#### Which issue(s) does this PR fixes?:

Fixes #

### CODE REVIEW

File: packages/jellyfish-api-core/__tests__/category/loan/getLoanInfo.test.ts [hunk 1]
```diff
@@ -558,4 +558,146 @@ describe('Loan - getLoanInfo', () => {
...
+    await testing.rpc.oracle.setOracleData(collateralOracleId2, now(), {
+      prices: [
+        { tokenAmount: '1@DFI', currency: 'USD' },
+        { tokenAmount: '10000@BTC', currency: 'USD' },
+        { tokenAmount: '101@ETH', currency: 'USD' },
+        { tokenAmount: '10@TSLA', currency: 'USD' },
+        { tokenAmount: '4@AMZN', currency: 'USD' }
+      ]
+    })
+    await testing.generate(10)
```
Comment (surangap) [inline packages/jellyfish-api-core/__tests__/category/loan/getLoanInfo.test.ts:588]:
```suggestion
```
you already have it in `beforeEach`, can use it

File: packages/jellyfish-api-core/__tests__/category/loan/getLoanInfo.test.ts [hunk 2]
```diff
@@ -558,4 +558,146 @@ describe('Loan - getLoanInfo', () => {
...
+    await testing.rpc.oracle.setOracleData(collateralOracleId2, now(), {
+      prices: [
+        { tokenAmount: '85@TSLA', currency: 'USD' }
+      ]
+    })
+    await testing.container.generate(12)
+
+    { // After
+      const auctions = await testing.container.call('listauctions')
+      expect(auctions.length).toStrictEqual(1)
```
Comment (surangap) [inline packages/jellyfish-api-core/__tests__/category/loan/getLoanInfo.test.ts:685]: Add a check for the `auctions.batchCount` also?

File: packages/jellyfish-api-core/__tests__/category/loan/getLoanInfo.test.ts [hunk 3]
```diff
@@ -558,4 +558,146 @@ describe('Loan - getLoanInfo', () => {
       })
     }
   })
+
+  it('should count openAuctions', async () => {
```
Comment (surangap) [inline packages/jellyfish-api-core/__tests__/category/loan/getLoanInfo.test.ts:562]:
```suggestion
  it('should check openAuctions to reflect auction batch count ', async () => {
```

File: packages/jellyfish-api-core/__tests__/category/loan/getLoanInfo.test.ts (hunk 1 above)
Comment (chanakasameera) [reply packages/jellyfish-api-core/__tests__/category/loan/getLoanInfo.test.ts]: A new oracle price feeder was required for this test case, therefore added a new one.

Comment (surangap) [reply packages/jellyfish-api-core/__tests__/category/loan/getLoanInfo.test.ts]: hmm. why do you need a new oracle? you could update the prices of the existing one?

**Review by surangap — DISMISSED** [review]

**Review by canonbrother — DISMISSED** [review]

File: packages/testcontainers/src/containers/DeFiDContainer.ts [hunk 4]
```diff
@@ -35,7 +35,7 @@ export abstract class DeFiDContainer extends DockerContainer {
     if (process?.env?.DEFICHAIN_DOCKER_IMAGE !== undefined) {
       return process.env.DEFICHAIN_DOCKER_IMAGE
     }
-    return 'defi/defichain:2.6.2'
+    return 'defi/defichain:HEAD-3a17844'
```
Comment (surangap) [inline packages/testcontainers/src/containers/DeFiDContainer.ts:38]:
```suggestion
    return 'defi/defichain:master-2419f511c'
```

**Review by fuxingloh — APPROVED** [review]

**Pull request merged**